catalog/project/kcc-namespace/kcc.yaml

```diff
@@ -12,65 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Enable KCC for this namespace
+apiVersion: core.cnrm.cloud.google.com/v1beta1
+kind: ConfigConnectorContext
+metadata:
+  name: configconnectorcontext.core.cnrm.cloud.google.com
+  namespace: project-id # {"$kpt-set":"project-namespace"}
+  annotations:
+    cnrm.cloud.google.com/blueprint: cnrm/kcc-namespace/v0.2.0-alpha.1
+spec:
+  googleServiceAccount: kcc-project-id@management-project-id.iam.gserviceaccount.com # {"$kpt-set":"project-id-sa-email"}
+---
 # Create GCP ServiceAccount for use by KCC to manage resources in this project
 apiVersion: iam.cnrm.cloud.google.com/v1beta1
 kind: IAMServiceAccount
 metadata:
   name: kcc-project-id # {"$kpt-set":"project-id-sa"}
   namespace: config-control # {"$kpt-set":"management-namespace"}
   annotations:
-    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:project/v0.2.0
     cnrm.cloud.google.com/project-id: management-project-id # {"$kpt-set":"management-project-id"}
+    cnrm.cloud.google.com/blueprint: cnrm/kcc-namespace/v0.2.0-alpha.1
 spec:
   displayName: kcc-project-id # {"$kpt-set":"project-id-sa"}
 ---
-# Grant KCC owner of this project
-apiVersion: iam.cnrm.cloud.google.com/v1beta1
-kind: IAMPolicyMember
-metadata:
-  name: kcc-project-id-owners-permissions # {"$kpt-set":"project-id-sa-owners-permissions"}
-  namespace: projects # {"$kpt-set":"projects-namespace"}
-  annotations:
-    cnrm.cloud.google.com/blueprint: cnrm/landing-zone:project/v0.2.0
-    cnrm.cloud.google.com/project-id: management-project-id # {"$kpt-set":"management-project-id"}
-spec:
-  member: "serviceAccount:kcc-project-id@management-project-id.iam.gserviceaccount.com" # {"$kpt-set":"project-id-sa-ref"}
-  resourceRef:
-    name: project-id # {"$kpt-set":"project-id"}
-    apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
-    kind: Project
-  role: roles/owner
----
 # Allow KCC's Kubernetes Service Account to use the GCP ServiceAccount
 apiVersion: iam.cnrm.cloud.google.com/v1beta1
 kind: IAMPolicyMember
 metadata:
+  annotations:
```

Comment:
Looks like a merge conflict or something. Line 45 probably needs to come here...

Reply:
>.< bad copy/paste.
kpt fmt wanted to move everything to be alphabetical and I had to fight it.
fixed now...